src/System.Management.Automation/engine/remoting/commands/InvokeCommandCommand.cs

```diff
@@ -54,7 +54,7 @@ namespace Microsoft.PowerShell.Commands
     /// Create a new runspace and use it to execute a command on a remote machine
     ///     $runspace = New-PSSession -computername "Server1"
     ///     $credential = get-credential "user01"
-    ///     invoke-command -command {get-process} �Crunspace $runspace -credential $credential
+    ///     invoke-command -command {get-process} �Crunspace $runspace -credential $credential
```

Comment:
VS Code again catch a special character 😕 Obviously this is considered to be `-runspace`
MS VS also corrupt this.
Can I fix it in the PR?